config/monitor_catalog_services.go

```diff
@@ -25,6 +25,10 @@ type CatalogServicesMonitorConfig struct {
 	DeprecatedSourceIncludesVar *bool `mapstructure:"source_includes_var"`
 }
 
+func (c *CatalogServicesMonitorConfig) VariableType() string {
+	return "catalog_services"
```

Comment:
I really like this approach to uniqueness! 

One thing, can we make this string a constant? Same for other variable types, then it would be easier to refer to them in other parts of code, test cases, etc. in the future

Reply:
could even use the existing catalogServicesType const, the other monitor's also include a const type variable

Reply:
It looks like `catalogServicesType` const is actually the condition type vs. variable type e.g. `catalog-services` vs. `catalog_services`. It's a little bit different so worried that reusing may cause confusion. Otherwise, it would be great to reuse it.

Right now, there's no code referring to the variable type content within the configuration package. I personally prefer to abstract constants once they're needed. There are other packages that could potentially use the constants, but I think we were also trying to limit how much of the config package we import until we refactor - I might be remembering this wrong.

Let me know your thoughts!

Reply:
ah ok, I didn't notice the difference

If you want to wait until the const is needed, I'm okay with leaving the consts out for now then.